src/lavinmq/auth/authenticators/rabbit_http_auth.cr

```diff
@@ -0,0 +1,41 @@
+require "../authenticator"
+require "../../server"
+
+module LavinMQ
+  module Auth
+    class RabbitBackendAuthenticator < Authenticator
```

Comment:
Also, the naming should be HTTPBackendAuthenticator? avoid using rabbit in naming :)

Reply:
Ok, I'm removing all mentions of Rabbit.